### PR TITLE
show opengl version & update image

### PR DESCRIPTION
**Pull request by SimonDanisch**

### CODE REVIEW

**Comment by asinghvi17** [comment]
Sorry for the mess I'm about to create, I needed a test PR that wasn't mine so this is it 😜

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -3,8 +3,9 @@ variables:
     MODERNGL_DEBUGGING: "true"
     JULIA_DEPOT_PATH: "$CI_PROJECT_DIR/.julia/"
 
+# It's broken so this is irrelevant anyway, we assume that the tests were successful
```
Comment (asinghvi17) [inline .gitlab-ci.yml]: delete this entirely

Comment (asinghvi17) [reply .gitlab-ci.yml]: @femtocleaner

**Comment by asinghvi17** [comment]
Is this ready for merge?

**Comment by asinghvi17** [comment]
What's with Gitlab?  Test isn't showing...

**Comment by SimonDanisch** [comment]
weird...

**Comment by asinghvi17** [comment]
> was by accidant?

I needed a PR that I didn't author to test if femtocleaner worked...it didn't.  I assume the server's down or good or something..

**Comment by SimonDanisch** [comment]
closed in favor of https://github.com/JuliaPlots/MakieGallery.jl/pull/68

Yeah server down could be

**Comment by asinghvi17** [comment]
Gitlab's website says everything is fine

**Pull request closed without merging**